constants/event_flags.asm

```diff
@@ -201,13 +202,9 @@
 	const EVENT_GAVE_GS_BALL_TO_KURT
 	const EVENT_FOREST_IS_RESTLESS
 	const EVENT_ANSWERED_DRAGON_MASTER_QUIZ_WRONG
-; Unused
-	const EVENT_0C2
-	const EVENT_0C3
-	const EVENT_0C4
-	const EVENT_0C5
-	const EVENT_0C6
-	const EVENT_0C7
+; Unused: $0C2-$0C7
```

Comment:
Is this useful at all? Maybe just document the size of the gap instead of the exact values?

Reply:
Either is fine with me. I guess the size is more intuitive; "you can add this many events within the gap".